lib/repo/git_hooks/multihook.py

```diff
@@ -11,7 +11,6 @@
     with open(os.path.join(real_dir, 'max_file_size')) as f:
         max_file_size = f.read().strip()
     hooks_dir = os.path.join(real_dir, '{}.d'.format(hook_type))
-    env = {**os.environ, 'MAX_FILE_SIZE': max_file_size}
```

Comment:
The `env` variable is used on line 19, so I'm not sure how this works?

For `subprocess.run` do we need to explicitly pass in the current environment variables if we want them to be used, or are they automatically passed if we don't specify an `env`?

Reply:
Sorry about this. I was working on the changes for this PR and the one for the hook rewrite PR at the same time so I dropped the ball on updating the hooks for this PR as needed. 

(if env=None then it inherits the parent process' environment)